### ISSUE TITLE
[Bug] Google maps plugin being a buggy mess

### ISSUE THREAD

**Issue by KhaiCo** [KieronQuinn/SmartspacerPlugins#9]
### Select the Plugin you are reporting for

Google Maps

### Description

# Issue
I'm on POCO F2 Pro running A12
So basically the Google maps plugin only works for me sometimes, as seen here:
![Screenshot_2023-11-05-12-19-47-441-edit_com kieronquinn app smartspacer](https://github.com/KieronQuinn/SmartspacerPlugins/assets/130490107/14d4922d-a9fa-4b49-8f58-f76cf0f71b6f)
^No requirements set
![Screenshot_2023-11-05-12-20-26-105-edit_app lawnchair debug](https://github.com/KieronQuinn/SmartspacerPlugins/assets/130490107/79275d02-c357-4081-910e-a718ca638ba5)
^still not working

And also the fact that it crashes when I opened the settings

https://github.com/KieronQuinn/SmartspacerPlugins/assets/130490107/c769a6af-08b0-421f-937a-6daf1fcf8162

## How I found this
Basically from the start I downloaded the plugin, it didn't work correctly and only showing "grant location permission" (which I have granted) but suddenly it worked but when I wanted to tweak the settings a bit the issue mentioned above appeared out of nowhere

## Log
There seems to be no logging options in the settings but I've turned on crash report

**Comment by KieronQuinn** [comment]
Add the Maps widget to your homescreen and follow the instructions on it. This isn't a plugin issue, it's maps not updating the widget.
